packages/leancode_lint/lib/lints/prefer_center_over_align.dart

```diff
@@ -1,141 +1,143 @@
-import 'package:analyzer/dart/analysis/results.dart';
-import 'package:analyzer/dart/ast/ast.dart';
-import 'package:analyzer/dart/constant/value.dart';
-import 'package:analyzer/diagnostic/diagnostic.dart';
-import 'package:analyzer/error/listener.dart';
-import 'package:analyzer_plugin/utilities/range_factory.dart';
-import 'package:custom_lint_builder/custom_lint_builder.dart';
-import 'package:leancode_lint/helpers.dart';
+// TODO: decide what to do with this lint in light of https://github.com/dart-lang/sdk/issues/61517
```

Comment:
What I gathered from the SDK team responses is that there is no good way to resolve this. I think we need to give up on computeConstantValue approach and just hardcode some cases. Or, I can try the proposed approach of crawling from the `Align` widget to the `Alignment.center` constant. I wonder how expensive this is.